test/integration/users.test.js

```diff
@@ -241,8 +241,8 @@ describe("Users", function () {
           expect(res.body).to.be.a("object");
           expect(res.body.message).to.equal("User returned successfully!");
           expect(res.body.user).to.be.a("object");
-          expect(res.body.user).to.not.have.property("phone");
-          expect(res.body.user).to.not.have.property("email");
+          // expect(res.body.user).to.not.have.property("phone");
+          // expect(res.body.user).to.not.have.property("email");
```

Comment:
@jalpa015 Any reason why we want to keep dead code?

Reply:
@ankushdharkar  I was unsure about making changes to tests file. So I left it commented. I will remove it.

Reply:
@ankushdharkar  Can I do something to fix the GithubSerive tests?